lib/app_server.py

```diff
@@ -118,7 +118,7 @@ def logfile(self):
         test_name = os.path.basename(self.current_test.name)
         # add :conf_name if any
```

Comment:
This comment is obsoleted by the change.

Reply:
Thanks! I didn't notice this line at all.
I decided to just update it by adding a new character to the comment.